### ISSUE TITLE
tokenUtxoDetails details reports UTXO a non-SLP UTXO when rate limited

### ISSUE THREAD

**Issue by christroutner** [Permissionless-Software-Foundation/bch-js#26]
The following is a UTXO that triggered this bug:

```
    {
      "utxos": [
        {
          "height": 654522,
          "tx_hash": "072a1e2c2d5f1309bf4eef7f88684e4ecd544a903b386b07f3e04b91b13d8af1",
          "tx_pos": 0,
          "value": 6999,
          "satoshis": 6999,
          "txid": "072a1e2c2d5f1309bf4eef7f88684e4ecd544a903b386b07f3e04b91b13d8af1",
          "vout": 0,
          "isValid": false
        },
        {
          "height": 654522,
          "tx_hash": "a72db6a0883ecb8e379f317231b2571e41e041b7b1107e3e54c2e0b3386ac6ca",
          "tx_pos": 1,
          "value": 546,
          "satoshis": 546,
          "txid": "a72db6a0883ecb8e379f317231b2571e41e041b7b1107e3e54c2e0b3386ac6ca",
          "vout": 1,
          "isValid": false
        }
      ],
      "address": "bitcoincash:qzygn28zpgeemnptkn26xzyuzzfu9l8f9vfvq7kptk"
    }
```

The first is a non-SLP UTXO. The second is a valid SLP UTXO. Here is the response from bch-api:

`{"message":"request: 172.17.0.4 - GET /v3/rawtransactions/getRawTransaction/a72db6a0883ecb8e379f317231b2571e41e041b7b1107e3e54c2e0b3386ac6ca?verbose=true 429 1.041 ms - 126 axios/0.19.0\n","level":"info","timestamp":"2020-09-28T04:30:03.631Z"}`

The `getRawTransaction()` part of the tokenUtxoDetails() call got a 429 response. As a result, the UTXO information that came back indicated that the UTXO `isValid` property as `false`. This is incorrect and could lead to burning tokens.

The proper return value for `isValid` should be `null` to indicate the level of uncertainty.

**Comment by christroutner** [comment]
The `getRawTransaction` call is originating from `decodeOpReturn()`. An error is thrown when calling `decodeOpReturn()` on a non-SLP token (isValid should be `false`), but an error is also thrown when getting a 429 error (isValid should be `null`).

Code needs to be added to the `decodeOpReturn()` function to detect if the status of the call was 200 or 429. If 200, then `isValid` can be marked `false`. If `429` then `isValid` should be marked `null`.

This needs to be captured in a unit test.

**Comment by christroutner** [comment]
This bug addressed by PR #27
